internal/db/migrations/2024-05-22.0-ingest_payments.sql

```diff
@@ -0,0 +1,34 @@
+-- +migrate Up
+
+CREATE TABLE ingest_store (
+  key varchar(255) NOT NULL,
+  value varchar(255) NOT NULL,
+  PRIMARY KEY (key)
+);
+
+CREATE TABLE ingest_payments (
+  operation_id bigint NOT NULL,
+  operation_type text NOT NULL,
+  transaction_id bigint NOT NULL,
```

Comment:
I'd name this field `stellar_transaction_id` to make it clear that it is from the network. Only a suggestion.